### PR TITLE
#4645 Add nodeNames definition to Shabondi API

### PR DESCRIPTION
**Pull request by vitojeng**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/ShabondiRoute.scala [hunk 1]
```diff
@@ -120,6 +120,10 @@ private[configurator] object ShabondiRoute {
         }
         throw new IllegalArgumentException(s"$key cannot be empty.")
       }
+      if (clusterInfo.nodeNames.size != 1) {
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/ShabondiRoute.scala:123]: Is this a TODO?

Comment (vitojeng) [reply ohara-configurator/src/main/scala/oharastream/ohara/configurator/route/ShabondiRoute.scala:123]: yep. I'll add a TODO description.

File: docs/src/rest/rest-stream.rst [hunk 2]
```diff
@@ -72,7 +72,6 @@ The following are common settings to a stream app.
      {"group": "default", "name": "n"}
 
 #. tags (**object**) — the user defined parameters
-#. nodeNames (**array(string)**) — node list of stream running container
```
Comment (vitojeng) [inline docs/src/rest/rest-stream.rst:75]: Remove duplicate item

**Comment by vitojeng** [comment]
not related error

**Pull request merged**